.gitignore

```diff
@@ -61,3 +61,6 @@ coverage.xml
 .coverage
 .vscode/tasks.json
 todo.txt
+
+tags
+.vimrc
```

Comment:
Could you remove this? I think this might be better suited for your [global .gitignore](https://docs.github.com/en/github/getting-started-with-github/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)

Reply:
Sure! Thanks for the link; learnt something new today